HGSdataDownload.m

```diff
@@ -58,7 +58,7 @@
     HGSdata = struct();
 end
 
-
+% Someone wants kill himself after this code

```

Comment:
Caleb please write the comment in positive, something like "This was hard to do" ?